.idea/codeStyles/Project.xml

```diff
@@ -325,4 +325,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
```

Comment:
I didnt add any new lines at the and of the file, I dont get what this change is. Any idea?